doc/changelog_update.md

```diff
@@ -32,8 +32,9 @@ bullet points might be helpful:
   need to select the Rust release tag from the dropdown and then check the
   commit of the Clippy directory:
 
-  ![Explanation of how to find the commit hash](https://user-images.githubusercontent.com/2042399/62846160-1f8b0480-bcce-11e9-9da8-7964ca034e7a.png)
-
+To find the commit hash, click on "History" of the relevant branch in github
+and search for the latest "Merge commit '<hash>' into <branch>" commit.
+The <hash> part is then the most recent commit in the clippy repo.
```

Comment:
Executing this command:

```bash
git log --oneline -- src/tools/clippy/ | grep -o "Merge commit '[a-f0-9]*' into .*" | head -1 | sed -e "s/Merge commit '\([a-f0-9]*\)' into .*/\1/g"
```

will output the correct hash, when executed in a `rust-lang/rust` checkout.

(This will not work, when the merge commit message was changed during the sync process, but I don't think, that we have to worry about this, so I wouldn't mention it here)